<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConfiguration</class>
 <widget class="QDialog" name="DialogConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icon/icon_configration</normaloff>:/icon/icon_configration</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Path</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Project Folder: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Mongodb:</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QxtLineEdit" name="lineEditMongodb">
          <property name="toolTip">
           <string>Mongodb文件夹
比如D:/blog/mongodb或../mongodb</string>
          </property>
          <property name="whatsThis">
           <string>Mongodb文件夹
比如D:/blog/mongodb或../mongodb</string>
          </property>
          <property name="text">
           <string>H:/blog/mongodb</string>
          </property>
          <property name="buttonAutoRaise">
           <bool>false</bool>
          </property>
          <property name="buttonIcon">
           <iconset resource="icons.qrc">
            <normaloff>:/icon/icon_folder_mongodb</normaloff>:/icon/icon_folder_mongodb</iconset>
          </property>
          <property name="resetButtonMode">
           <enum>QxtLineEdit::ShowResetNotEmpty</enum>
          </property>
          <property name="sampleText">
           <string>Example: D:/blog/mongodb or ../db</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QxtLineEdit" name="lineEditProjectFolder">
          <property name="toolTip">
           <string>利用Express框架构建的Node项目文件夹
比如D:/blog或../</string>
          </property>
          <property name="whatsThis">
           <string>利用Express框架构建的Node项目文件夹
比如D:/blog或../</string>
          </property>
          <property name="text">
           <string notr="true">H:/blog</string>
          </property>
          <property name="buttonAutoRaise">
           <bool>false</bool>
          </property>
          <property name="buttonIcon">
           <iconset resource="icons.qrc">
            <normaloff>:/icon/icon_folder_projectfolder</normaloff>:/icon/icon_folder_projectfolder</iconset>
          </property>
          <property name="resetButtonMode">
           <enum>QxtLineEdit::ShowResetNotEmpty</enum>
          </property>
          <property name="sampleText">
           <string>Example: D:/blog or ../</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButtonProjectFolder">
          <property name="text">
           <string>Select...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icon/icon_folder_projectfolder</normaloff>:/icon/icon_folder_projectfolder</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButtonMongodb">
          <property name="text">
           <string>Select...</string>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icon/icon_folder_mongodb</normaloff>:/icon/icon_folder_mongodb</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Shortcut Keys</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <item row="1" column="1">
       <widget class="QKeySequenceEdit" name="keySequenceEditStart"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Start Service:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Stop Service:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QKeySequenceEdit" name="keySequenceEditStop"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Restart Service:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QKeySequenceEdit" name="keySequenceEditRestart"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Service</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxMongodb">
        <property name="text">
         <string>Mongo DB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxNode">
        <property name="text">
         <string>Node</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icon/icon_check</normaloff>:/icon/icon_check</iconset>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonLoadDefault">
       <property name="text">
        <string>Load Default</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/icon/icon_reset</normaloff>:/icon/icon_reset</iconset>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QxtLineEdit</class>
   <extends>QLineEdit</extends>
   <header>QxtLineEdit</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
